server/src/data_worker.rs

```diff
@@ -274,7 +267,9 @@ impl DataWorker {
             // listen for Reload order
             let has_reload_message = reload_consumer.next().await;
             info!("Received a message on the reload queue.");
-            self.handle_reload_message(has_reload_message, channel)
+            // do not reload realtime here, because the realtime queue is not created yet, so we may
+            // loose some realtime messages
```

Comment:
Not sure: shouldn't this be `lose`? (https://www.merriam-webster.com/words-at-play/lose-vs-loose-usage)

```suggestion
            // lose some realtime messages
```